src/components/Autocomplete.vue

```diff
@@ -21,11 +21,11 @@
 </template>
 
 <script>
-import algoliaComponent from '../mixins/component';
+import { createAlgoliaMixin } from '../mixins/component';
```

Comment:
We can find another name that don't mention Algolia like `createWidgetMixon` or something else? We can also rename the file with the name found.